cli/productutil.py

```diff
@@ -37,7 +37,7 @@ def upload_product_images(
     protecode_cfg_name: str,
     product_cfg_file: CliHints.existing_file(),
     processing_mode: CliHint(
-        choices=['upload_if_changed', 'rescan']
+        choices=['upload_if_changed', 'rescan', 'force_upload']
```

Comment:
this is much too cumbersome. I replaced this:

https://github.com/gardener/cc-utils/commit/d3c6d5f0245f7ae52d36f0ebabe6e9c05ac3c222